client/components/RoomListItem.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const RoomListItem = React.createClass({
+  mixins: [ReactMeteorData],
+
+  getMeteorData () {
+    const messagesHandle = Meteor.subscribe("chatterMessages", {
+      roomId: this.props.room._id,
```

Comment:
this.props.room._id repeats many times, use const roomId and get the benefit of { roomId } syntax